mailbox/store/src/test/java/org/apache/james/mailbox/store/AbstractCombinationManagerTest.java

```diff
@@ -384,7 +384,9 @@ public void setInMailboxesFromMessageIdManagerShouldMoveMessage() throws Excepti
         assertThat(messageManager1.getMessages(MessageRange.all(), FetchGroupImpl.MINIMAL, session)).isEmpty();
         assertThat(messageManager2.getMessages(MessageRange.all(), FetchGroupImpl.MINIMAL, session))
             .hasSize(1)
-            .extractingResultOf("getMessageId").containsOnly(messageId);
+            .extractingResultOf("messageMetaData")
```

Comment:
why don't you use method references instead of some strings ?

Reply:
I guess it is a Java 6 legacy...